cmd/kubeadm/app/preflight/checks.go

```diff
@@ -892,13 +892,13 @@ func RunInitMasterChecks(execer utilsexec.Interface, cfg *kubeadmapi.MasterConfi
 	if cfg.Etcd.External != nil {
 		// Only check etcd version when external endpoints are specified
 		if cfg.Etcd.External.CAFile != "" {
-			checks = append(checks, FileExistingCheck{Path: cfg.Etcd.External.CAFile})
+			checks = append(checks, FileExistingCheck{Path: cfg.Etcd.External.CAFile, Label: "ExternalEtcdClientCertificates"})
```

Comment:
Better have difference labels to avoid confusions. WDYT?

Reply:
It was original reason discussed in kubernetes/website#8317: for user in case some issue with client certificates to specify one type in `--ignore-preflight-errors` rather than 3 or 4 different. Otherwise user would do `--ignore-preflight-errors=all`.

Reply:
Okay, fair enough. Thanks!